src/fragments/lib/push-notifications/react-native/migrate_from_previous_version/migrate-from-previous-version.mdx

```diff
@@ -0,0 +1,170 @@
+
+```ruby
+platform :ios, '13.0'
+```
+
+3. Install the new pods by running the following command at the root of your React Native project.
+
+```bash
+npx pod-install
+```
+
+### Update your Application Delegate
+
+1. Reverse the steps previously taken to [augment your `AppDelegate`](https://github.com/react-native-push-notification/ios#augment-appdelegate)
+
+2. Locate and open your `AppDelegate.m` or `AppDelegate.mm` file in your text editor. You should find it in your React Native project under `/ios/<your-project-name>`
+
+3. At the top of your `AppDelegate`, import Amplify Push Notifications:
+
+```objective-c
+#import "AppDelegate.h"
+#import "AmplifyPushNotification.h"
+
+...
+```
+
+4. In the body of your `AppDelegate`, add the following two methods:
+
+```objective-c
+- (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken {
+  [AmplifyPushNotification didRegisterForRemoteNotificationsWithDeviceToken:deviceToken];
+}
+
+- (void)application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo fetchCompletionHandler:(void (^)(UIBackgroundFetchResult result))completionHandler {
+  [AmplifyPushNotification didReceiveRemoteNotification:userInfo withCompletionHandler:completionHandler];
+}
+```
+
+</Block>
+<Block name="Android">
+
+### Update gradle scripts
+
+Amplify Push Notifications now requires a minimum target of Android SDK API level 24.
+
+1. Open `build.gradle` located inside the `/android` folder of your React Native project with a text editor.
+
+2. Update the `minSdkVersion` value in the `buildscript` block to at least `24`.
+
+```groovy
+buildscript {
+  ext {
+    ...
+    minSdkVersion = 24
+    ...
+  }
+}
+```
+
+3. Open the application `build.gradle` located inside the `/android/app` folder of your React Native project with a text editor.
+
+4. Remove firebase dependencies from the `dependencies` block.
+
+```diff
+dependencies {
+  ...
+-   implementation "com.google.firebase:firebase-core:15.0.2"
+-   implementation "com.google.firebase:firebase-messaging:15.0.2"
+  ...
+}
+```
+
+### Update the manifest
+
+5. Open `AndroidManifest.xml` located inside the `/android/app/src/main` folder.
+
+6. Remove the old service block from the `application`.
+
+```diff
+<application ... >
+  ...
+- <!--[START Push notification config -->
+- <service
+-   android:name="com.amazonaws.amplify.pushnotification.RNPushNotificationMessagingService"
+-   android:exported="false">
+-   <intent-filter>
+-     <action android:name="com.google.firebase.MESSAGING_EVENT"/>
+-   </intent-filter>
+- </service>
+- <!-- [END Push notification config -->
+  ...
+</application>
+```
+
+</Block>
+</BlockSwitcher>
+
+## Update your application code
+
+### Add required polyfills
+
+You need to add the crypto.getRandomValues and URL polyfill to your application's entry point file (in most React Native apps this will be the top level `index.js`).
```

Comment:
```suggestion
You need to add the crypto.getRandomValues and URL polyfills to your application's entry point file (in most React Native apps this will be the top level `index.js`).
```